packages/edit-site/src/components/editor/index.js

```diff
@@ -207,6 +207,7 @@ export default function EditSiteEditor( { isLoading } ) {
 					extraSidebarPanels={
 						! isEditingPage && <PluginTemplateSettingPanel.Slot />
 					}
+					isEditMode={ isEditMode }
```

Comment:
Let's avoid this prop please. "edit mode" doesn't really make sense in the editor package.

Reply:
Of course. I wasn't aware that we have this info in `isPreviewMode`. Updated.